__tests__/screens/BackupWords.test.tsx

```
@@ -0,0 +1,210 @@
+    await waitFor(() => {
+      expect(queryByText('Wrong word.')).toBeTruthy();
+    });
+  });
+
+  it('completes all 5 steps and shows success modal', async () => {
+    const { getByText, queryByText } = render(
+      <BackupWords navigation={mockNavigation} route={mockRoute} />
+    );
+
+    // Select the correct word for each of the 5 steps
+    for (let i = 0; i < 5; i++) {
```

ESLint

[error] Unary operator '++' used.

(no-plusplus)